docs/content/preview/yugabyte-platform/_index.md

```diff
@@ -24,30 +29,169 @@ You can access YugabyteDB Anywhere via an Internet browser that has been support
 
 YugabyteDB Anywhere offers three levels of user accounts: Super Admin, Admin, and Read-only, with the latter having rather limited access to functionality. Unless otherwise specified, the YugabyteDB Anywhere documentation describes the functionality available to a Super Admin user.
 
-### Get started
-
-<div class="row">
-  <div class="col-12 col-md-6 col-lg-12 col-xl-6">
-    <a class="section-link icon-offset" href="install-yugabyte-platform/install/public-cloud/">
-      <div class="head">
-        <img class="icon" src="/images/section_icons/quick_start/install.png" aria-hidden="true" />
-        <div class="title">Install</div>
+<div class="row cloud-laptop">
+  <div class="col-12 col-md-12 col-lg-6">
+    <div class="border two-side">
+      <div class="body">
+        <div class="box-top">
+          <span>Try YugabyteDB Anywhere</span>
+        </div>
+        <div class="body-content">Request a free 30-day trial of YugabyteDB Anywhere. No credit card required.</div>
+        <a class="text-link" href="install-yugabyte-platform/" title="Learn more">Learn more</a>
+      </div>
+      <div class="image">
+        <img class="icon" src="/images/homepage/yugabyte-in-cloud.png" title="Yugabyte cloud" aria-hidden="true">
       </div>
+    </div>
+  </div>
+  <div class="col-12 col-md-12 col-lg-6">
+    <div class="border two-side">
       <div class="body">
-        Installing YugabyteDB Anywhere in any environment
+        <div class="box-top">
+          <span>Install YugabyteDB Anywhere</span>
+        </div>
+        <div class="body-content">Install on any environment, including Kubernetes, public cloud, or private cloud.</div>
+        <a class="text-link" href="install-yugabyte-platform/prerequisites/" title="Learn more">Learn more</a>
+      </div>
+      <div class="image">
+        <img class="icon" src="/images/homepage/locally-laptop.png" title="Locally Laptop" aria-hidden="true">
       </div>
-    </a>
+    </div>
   </div>
+</div>
 
-  <div class="col-12 col-md-6 col-lg-12 col-xl-6">
-    <a class="section-link icon-offset" href="configure-yugabyte-platform/configure/aws/">
-      <div class="head">
-        <img class="icon" src="/images/section_icons/quick_start/create_cluster.png" aria-hidden="true" />
-        <div class="title">Configure</div>
+<div class="three-box-row">
```

Comment:
Think I would put Install and Configure in top boxes, and trim this section to just deploy, manage, backup, secure - topics where you are dealing with deployments (as opposed to YBA itself)